x-pack/metricbeat/module/oracle/tablespace/_meta/data.json

```diff
@@ -0,0 +1,43 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "event": {
+        "dataset": "oracle.tablespace",
+        "duration": 115000,
+        "module": "oracle"
+    },
+    "metricset": {
+        "name": "tablespace"
+    },
+    "oracle": {
+        "tablespace": {
+            "data_file": {
+                "id": 17,
+                "name": "/u02/app/oracle/oradata/ORCLCDB/orclpdb1/system01.dbf",
+                "online_status": "SYSTEM",
+                "size": {
+                    "bytes": 283115520,
+                    "free": {
+                        "bytes": 282066944
+                    },
+                    "max": {
+                        "bytes": 34359721984
+                    }
+                },
+                "status": "AVAILABLE"
+            },
+            "name": "SYSTEM",
+            "space": {
+                "free": {
+                    "bytes": 3211264
+                },
+                "used": {
+                    "bytes": 283115520
```

Comment:
Nope :slightly_smiling_face: but the fact that it wasn't inmediately clear its purpose makes me think that a better naming is needed. It's actually a very subtle detail because it's indented at the same level higher than the `data_file` key. `used.bytes` is the size of the entire Tablespace, which could be many data files. `data_file.size.bytes` is the size of that particular file.